source/includes/usage-examples/sample-app.kt

```diff
@@ -0,0 +1,22 @@
+import com.mongodb.ConnectionString
+import com.mongodb.MongoClientSettings
+import com.mongodb.kotlin.client.*
+import com.mongodb.client.model.Filters.*
+
+fun main() {
+    val uri = "<connection string URI>"
+    
+    val settings = MongoClientSettings.builder()
+        .applyConnectionString(ConnectionString(uri))
+        .retryWrites(true)
+        .build()
+
+    // Create a new client and connect to the server
+    val mongoClient = MongoClient.create(settings)
+    val database = mongoClient.getDatabase("<database name>")
+    val collection = database.getCollection<Movie>("<collection name>")
```

Comment:
I: if you are parametrizing the collection with the `Movie` class, you need to include the data class definition in this example as well.

Reply:
Now that you point it out, I think this should instead be made more generic so it can apply to whatever collection the user desires (since this example template shouldn't be specific to the movie sample database).